client/src/components/application/Examine/CompName.vue

```diff
@@ -97,7 +45,89 @@
+                    @click="holdRequest()"><u>H</u>old</button>
+
+            <!-- CANCEL button -->
+            <button class="btn btn-sm btn-danger p2" id="examine-cancel-button"
+                    v-if="!is_making_decision && !is_cancelled && !is_approved_expired && !is_consumed" data-toggle="modal" data-target="#add-cancel-comment-modal">
+              Cancel Request</button>
+
+             <!-- EXAMINE button - to claim/examine an NR that is on hold -->
+            <button class="btn btn-sm btn-primary p2" id="examine-button" v-if="can_claim"
+                    @click="claimNR()" >Examine</button>
 
+              <!-- DECISION button -->
+            <button v-shortkey="['alt', 'd']" @shortkey="startDecision()" class="btn btn-sm btn-primary" id="examine-decide-button"
+                    v-if="!is_making_decision && !is_complete && is_my_current_nr"
+                    @click="startDecision()"><u>D</u>ecision</button>
+
+
+              <!-- RE-OPEN (un-furnished) button -->
+            <button class="btn btn-sm btn-danger p2" id="examine-re-open-button"
+                    v-if="is_complete && !is_furnished && !is_cancelled && !is_approved_expired" @click="reOpen()" >
+              Re-Open</button>
+
+            <!-- RESET (from furnished) button -->
+            <button class="btn btn-sm btn-danger p2" id="examine-reset-button"
+                    v-if="is_complete && is_furnished && !is_cancelled && !is_approved_expired" @click="reset()">
+              RESET</button>
+
+            <!-- ACCEPT/REJECT/CANCEL DECISION buttons -->
+            <button v-if="is_making_decision" v-shortkey="['alt', 'a']" @shortkey="nameAccept()" class="btn btn-sm btn-primary p2" id="decision-approve-button"
+                       @click="nameAccept()">
+                <u>A</u>pprove
+                </button>
+
+            <button v-shortkey="['alt', 'r']" @shortkey="nameReject()" class="btn btn-sm btn-danger p2" id="decision-reject-button"
+                    v-if="is_making_decision" @click="nameReject()" ><u>R</u>eject
+            </button>
+
+             <button v-shortkey="['alt', 'c']" @shortkey="is_making_decision=false" class="btn btn-sm btn-secondary p2 back-button " id="decision-cancel-button"
+                    v-if="is_making_decision" @click="is_making_decision=false">Ba<u>c</u>k
+            </button>
+
+
+          </div>
+          <div v-if="is_making_decision"  class="d-flex flex-row-reverse">
+
+            <div class="condition-checkbox-group p2">
+                 <b-form-checkbox
+                      class="condition-checkbox"
+                      v-model="consent_required"
+                 >
+                Include Consent / Condition
+                </b-form-checkbox>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+          <div v-if="!is_making_decision && !is_complete" id="manual-search">
```

Comment:
I didn't delete this... I moved it here so that it would still be in the right spot when using flex intead of float.